stl/inc/algorithm

```diff
@@ -11315,19 +11324,25 @@ namespace ranges {
             using _Vty = range_value_t<_Rng>;
 #if _VECTORIZED_FOR_X64_X86
             if constexpr (is_same_v<_Pj, identity> && sized_sentinel_for<decltype(_ULast), decltype(_UFirst)>) {
+#if _VECTORIZED_MINMAX
                 if constexpr (_Is_min_max_value_optimization_safe<decltype(_UFirst), _Pr>) {
                     if (!_STD is_constant_evaluated()) {
                         const auto _First_ptr = _STD to_address(_UFirst);
                         const auto _Last_ptr  = _First_ptr + static_cast<ptrdiff_t>(_ULast - _UFirst);
                         const auto _Result    = _STD _Minmax_vectorized(_First_ptr, _Last_ptr);
                         return {static_cast<_Vty>(_Result._Min), static_cast<_Vty>(_Result._Max)};
                     }
-                } else if constexpr (_Is_min_max_optimization_safe<decltype(_UFirst), _Pr>) {
-                    if (!_STD is_constant_evaluated()) {
-                        const auto _First_ptr = _STD to_address(_UFirst);
-                        const auto _Last_ptr  = _First_ptr + static_cast<ptrdiff_t>(_ULast - _UFirst);
-                        const auto _Result    = _STD _Minmax_element_vectorized(_First_ptr, _Last_ptr);
-                        return {*static_cast<const _Vty*>(_Result.first), *static_cast<const _Vty*>(_Result.second)};
+                } else
+#endif // ^^^ _VECTORIZED_MINMAX ^^^
+                {
+                    if constexpr (_Is_min_max_optimization_safe<decltype(_UFirst), _Pr>) {
+                        if (!_STD is_constant_evaluated()) {
+                            const auto _First_ptr = _STD to_address(_UFirst);
+                            const auto _Last_ptr  = _First_ptr + static_cast<ptrdiff_t>(_ULast - _UFirst);
+                            const auto _Result    = _STD _Minmax_element_vectorized(_First_ptr, _Last_ptr);
+                            return {
+                                *static_cast<const _Vty*>(_Result.first), *static_cast<const _Vty*>(_Result.second)};
+                        }

```

Comment:
I don't have enough context to understand why previously it was 'ok' for us to have this if-else chain without any macros, and now we need to 'erase' the body of the one of cases depending on `_VECTORIZED_MINMAX`. Can you please elaborate on what's changed here?